src/layouts/docs.tsx

```diff
@@ -206,7 +206,7 @@ interface DocsLayoutProps {
 }
 
 const DocsLayout: React.SFC<DocsLayoutProps> = ({canonical, title, body , arrows}) => (
-    <IndexLayout canonical={canonical} title={title}>
+    <IndexLayout canonical={canonical} title={title} description="The documentation of Gitpod, the online IDE for GitHub and GitLab.">
```

Comment:
Not quite sure about this description. Maybe with an extra word?

```suggestion
    <IndexLayout canonical={canonical} title={title} description="The documentation site of Gitpod, the online IDE for GitHub and GitLab.">
```